src/ui-test/tests/02_camel.settings.test.ts

```diff
@@ -63,13 +62,11 @@ describe('Camel User Settings', function () {
         it(`Should use '${customCamelVersion}' user defined Camel version`, async function () {
             await setCamelVersion(customCamelVersion);
             await executeCommand(CAMEL_RUN_ACTION_LABEL);
+            
+            // Static wait due initial camel instalations through the jbang
+            await driver.sleep(70000);
```

Comment:
🙁

Reply:
is this really needed? that is not nice at all 🙁

Reply:
Not needed, but it always will be (the first camel jbang run installs a lot of things and just wait for the response for 30 seconds) and dynamic wait with getText will run +-35 times with the negative result, the idea was to limit the call of getText() for now

To see it locally you need to clean jbang cache:

1) `jbang cache clear -o --all`
2) run some dap command

Reply:
deleted for now, but if we will face that "limit" issue again (in case no fix on extester side) it will be the first place to optimize :+1: